### ISSUE TITLE
Comptibility issue with the new Tidy 5e sheet

### ISSUE THREAD

**Issue by mech-tools** [slcao2/FoundryVTT-QuickRolls#61]
Hello,

There is a new release for Tidy 5e sheet that changes quite a lot of things.
I noticed that abilities rolls are no longer working.
Didn't check everything in the new sheet.

**Comment by slcao2** [comment]
@DarKDinDoN Oh fancy the new sheet looks pretty slick. I don't see the issue you're seeing however. The rolls are working for me. Can you post a screenshot of what happens when you roll?

**Comment by mech-tools** [comment]
Clicking on abilities or saves show this window instead of rolling
![image](https://user-images.githubusercontent.com/1687854/103488236-32dec880-4e0b-11eb-8b7c-ab3cc6f8fed5.png)

**Comment by slcao2** [comment]
Hmm can you send me a screenshot of the mods you have installed?

**Comment by mech-tools** [comment]
Yes, found it. This option from DAE (https://gitlab.com/tposney/dae):
![image](https://user-images.githubusercontent.com/1687854/103490050-74767000-4e19-11eb-906c-eefa2f48c0fe.png)

This allows the module to add or remove bonuses on saves


EDIT: same for skills & saves
